vadim.martsun/task_4/src/main/java/config/AppConfig.java

```diff
@@ -0,0 +1,35 @@
+package config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import random.RandomBankAccountService;
+import random.RandomBankAccountServiceImpl;
+import service.BankService;
+import service.BankServiceImpl;
+import service.TestBankService;
+import service.transactionExplorer.TransactionExplorer;
+import service.transactionExplorer.TransactionExplorerImpl;
+
+@Configuration
+public class AppConfig {
+
+    @Bean(name = "bankService")
+    public BankService bankService(){
+        return new BankServiceImpl();
+    }
+
+    @Bean(name = "testBankService")
+    public BankService testBankService(){
+        return new TestBankService();
+    }
+
+    @Bean(name = "randomBankAccountService")
+    public RandomBankAccountService randomBankAccountService(){
+        return new RandomBankAccountServiceImpl();
+    }
+
+    @Bean(name = "transactionExplorer")
```

Comment:
The same - you already have @Service annotation
You only need to use @ComponentScan for base package